packages/interface/src/components/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as PluginComplementaryArea } from './plugin-complementary-area';
+export { default as PluginComplementaryAreaHeader } from './plugin-complementary-area-header';
+export { default as PinnedItems } from './pinned-items';
```

Comment:
Now that is going to be exposed for public usage, should we prefix it with the plugin keyword? `PluginPinnedItems` or `PinnedPluginItems`?

Reply:
Even for PluginComplementaryArea, and PluginComplementaryAreaHeader, I'm not sure we should include plugin prefix, we are using these components our selves in the core. Maybe we can just discard all the plugin prefixes?

Reply:
Right, it might work this way as well 👍

Reply:
Names were updated.